src/components/workspace-utils.js

```diff
@@ -340,6 +346,55 @@ export const WorkspaceTagSelect = props => {
   })
 }
 
+export const WorkspaceSubmissionStatusIcon = ({ status, loadingSubmissionStats, size = 20 }) => {
+  return div({}, [
+    loadingSubmissionStats && h(DelayedRender, [
+      h(TooltipTrigger, {
+        content: 'Loading submission status',
+        side: 'left'
+      }, [spinner({ size })])
+    ]),
+    !!status && h(TooltipTrigger, {
+      content: span(['Last submitted workflow status: ', span({ style: { fontWeight: 600 } }, [_.startCase(status)])]),
+      side: 'left'
+    }, [
+      Utils.switchCase(status,
+        ['success', () => icon('success-standard', { size, style: { color: colors.success() } })],
+        ['failure', () => icon('error-standard', { size, style: { color: colors.danger(0.85) } })],
+        ['running', () => icon('sync', { size, style: { color: colors.success() } })]
+      )
+    ])
+  ])
+}
+
+export const RecentlyViewedWorkspaceCard = ({ workspace, submissionStatus, loadingSubmissionStats, timestamp }) => {
+  const { workspace: { namespace, name, googleProject } } = workspace
+
+  const dateViewed = Utils.makeCompleteDate(new Date(parseInt(timestamp)).toString())
+
+  return h(Clickable, {
+    style: { ...Style.elements.card.container, margin: '0 0.25rem 0 0.25rem', lineHeight: '22px', flex: '0 1 calc(25% - 10px)' },
+    href: Nav.getLink('workspace-dashboard', { namespace, name })
+  }, [
+    div({ style: { flex: 'none' } }, [
+      div({ style: { color: colors.accent(), ...Style.noWrapEllipsis, fontSize: 16, marginBottom: 7 } }, name),
+      div({ style: { display: 'flex', justifyContent: 'space-between' } }, [
+        div({ style: { ...Style.noWrapEllipsis, whiteSpace: 'pre-wrap', fontStyle: 'italic' } }, `Viewed ${dateViewed}`),
+        div({ style: { display: 'flex', alignItems: 'center' } }, [
+          h(WorkspaceSubmissionStatusIcon, {
```

Comment:
This WorkflowSubmissionStatusIcon is not accessible-- there seems to be no way to get the screen reader to announce anything about it.

Reply:
I also called this out in one of our tickets (https://broadworkbench.atlassian.net/browse/WOR-149?focusedCommentId=64541), so if you didn't actually change `WorkflowSubmissionStatusIcon`, that fix could be considered out of scope for this PR.

Reply:
This was just extraction, so I will leave it be for this PR